Modules/_io/textio.c

```diff
@@ -9,6 +9,7 @@
 #include "Python.h"
 #include "pycore_call.h"          // _PyObject_CallMethod()
 #include "pycore_codecs.h"        // _PyCodecInfo_GetIncrementalDecoder()
+#include "pycore_critical_section.h"
```

Comment:
Please leave a comment as the same as the other header. `// Py_BEGIN_CRITICAL_SECTION `

Reply:
Updated